integration/sql/iface-java/src/java/io/openlineage/sql/OpenLineageSql.java

```diff
@@ -0,0 +1,61 @@
+/*
+/* Copyright 2018-2022 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.sql;
+
+import org.apache.commons.lang3.SystemUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.file.Files;
+import java.util.List;
+
+public final class OpenLineageSql {
+    public static native SqlMeta parse(List<String> sql, String dialect, String defaultSchema) throws RuntimeException;
```

Comment:
Would it make sense to return Optional<SqlMeta> while leaving all the exceptions within this class. 
There is little we can do with them in other class.

Reply:
This is `native` method and it's not fun to change it's signature as it requires to change deep bowels of Rust code here https://github.com/OpenLineage/OpenLineage/blob/337e5c719580ad3ac40874f723808d8d9ebad403/integration/sql/iface-java/src/lib.rs#L13. 

We can have wrapper methods returning Optional easily though.